### ISSUE TITLE
Allow initial parameter values to be passed to dixoncoles

### ISSUE THREAD

**Issue by Torvaney** [Torvaney/regista#9]
Might be useful/faster when refitting values over time?

Something like:

```
fit <- dixoncoles_ext(hgoal ~ off(home) + def(away) + hfa + 0,
                      agoal ~ off(home) + def(home) + 0,
                      weight = ~ some_weighting(column_name)
                      data = games,
                      init = rep(0, 20))
```

+ refitting of the model with `purrr::accumulate`

**Comment by Torvaney** [comment]
Closed by 5fe9f5a268900b86c3125e2d037a669cf80e12b5

**Comment by Torvaney** [comment]
Closed by 5fe9f5a268900b86c3125e2d037a669cf80e12b5
